### DISCUSSION TITLE
zigbee2tasmota and sonoff snzb-03 sensor

### DISCUSSION THREAD

**Discussion by costascontis** [arendst/Tasmota#13279] [Q&A] [answered]
hi ,i flashed a sonoff zigbee bridge with tasmota and using it to bind 3 snzb-03 sonoff motion sensors.They work just fine reporting motion through mqtt.I need also to report me their battery status.I remember when i used the same sensors with OpenHab zigbee binding and a zigbee usb coordinator,the sensors report a battery low switch.With zigbee2tasmota i only can take battery percentage (number).The log report is 
`tele/zbBridge/Bath_sensor2/SENSOR = {"ZbReceived":{"Bath_sensor2":{"Device":"0xCC74","Name":"Bath_sensor2","BatteryVoltage":3.1,"BatteryPercentage":100,"Endpoint":1,"LinkQuality":55}}}`
I also saw the same sensor reports a batterylow switch with zigbee2mqtt .Is it possible this with zigbee2 tasmota?

**Comment by s-hadinger** [comment, marked as answer]
Zigbee2Tasmota reports what the device actually sends. There no such thing as battery low in Zigbee. You can interpret yourself the battery low threshold if percentage goes below a certain level.

**Reply by costascontis** [reply]
thnx for ur answer mate,I just refer to [this](https://www.zigbee2mqtt.io/devices/SNZB-03.html) that says that the sensor reports a batterylow switch but i guess its only for zigbee2mqtt ...Anyway do u have any insight or an example how to make a rule so when a battery goes under a level then send a mqtt topic with a switch?
